Makefile

```diff
@@ -269,9 +269,11 @@ run-metro: export TARGET := clojure
 run-metro: ##@run Start Metro to build React Native changes
 	@scripts/run-metro.sh
 
+export RE_FRISK_PORT ?= 4567
```

Comment:
This port is also used in the target `android-ports`. Could you update references to `4567` to this var?